.github/workflows/coverage.yml

```diff
@@ -7,15 +7,25 @@ on:
     branches: [ master ]
 jobs:
   coverage:
+
+    strategy:
+      matrix:
+        include:
+          - toolchain: nightly
+            timeout-minutes: 20
+          - toolchain: nightly-2024-08-29
+    timeout-minutes: ${{ matrix.timeout-minutes }}
```

Comment:
This might not run because `timeout-minutes` is unset in one of the matrix configs. Maybe just set `timeout-minutes: 20` for all coverage jobs for now, and add a comment to remove it when nightly is fixed?

Reply:
ah, thanks. I was thinking that might've been how I voided the whole workflow.